tax-integrations.yaml

```
@@ -0,0 +1,13 @@
+tax_integrations:
+  - tax_integration:
+      code: 'tax-integration-1'
+      title: 'My tax integration enabled'
+      active: true
+      stores:
+        - default
+  - tax_integration:
+      code: 'tax-integration-2'
+      title: 'My tax integration disabled'
+      active: false
+      stores:
+        - default
```

Do you mind if we just leave only one tax integration example? This is because tax is designed to be only one active at the moment, and as it's not possible to delete it with Rest endpoint, I'd keep it really simple to minimize what it does.
```suggestion
  - tax_integration:
      code: 'oope-tax'
      title: 'My tax integration'
      active: true
      stores:
        - default
```